src/views/ClickingGame.vue

```diff
@@ -0,0 +1,114 @@
+<template>
```

Comment:
Maybe you could use Vuetify in the future, instead of styling by hand?

Reply:
Yes! I will change it once I get the basic functionality done. Thanks kendrick :)